bukuserver/templates/bukuserver/statistic.html

```diff
@@ -55,26 +70,24 @@ <h4 class="modal-title" id="myModalLabel">Netloc ranking</h4>
         </div>
         <div class="modal-body">
           <table class="table table-condensed">
-            <tr>
-              <th>Rank</th>
-              <th>Netloc</th>
-              <th>Number</th>
-            </tr>
-            {% for item, number in netloc_counter.most_common() %}
-              {% if number > 1 %}
+            <thead class="bg-primary">
               <tr>
-                <td>{{loop.index}}</td>
+                <th>Rank</th>
+                <th>Netloc</th>
+                <th class="text-right">Number</th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for name, amount in netlocs.all %}
+              <tr>
+                <td>{{ loop.index }}</td>
                 <td>
-                  {% if item %}
-                  <a href="{{buku.filter('url_netloc_match', item)}}">{{item}}</a>
-                  {% else %}
-                  <span class="btn btn-default" disabled="disabled">(No Netloc)</span>
-                  {% endif %}
+                  <a href="{{ buku.filter('url_netloc_match', name) }}">{{ name or '(no netloc)' }}</a>
```

Comment:
There's no reason to disallow searching for entries with no netloc :sweat_smile: